src/vivarium_nih_us_cvd/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.dev353+gb79e496.d20231016"
```

Comment:
We don't want to commit this file

Reply:
@mattkappel  I was wondering about that. HOw does this all work w/o committing it?

Reply:
And do we add this to the .gitignore then?